app/assets/javascripts/discourse/app/services/composer.js

```diff
@@ -1596,7 +1596,7 @@ export default class ComposerService extends Service {
 
     // The two custom properties below can be overridden by themes/plugins to set different default composer heights.
     if (this.model.action === "reply") {
-      return "var(--reply-composer-height, 300px)";
+      return "var(--reply-composer-height, 255px)";
```

Comment:
Minor change to make sure the minimum height of the composer matches the `min-height` value in CSS.